microsoft/testsuites/core/timesync.py

```diff
@@ -151,3 +153,74 @@ def timesync_check_clocksource(self, node: Node) -> None:
                     f"{current_clock_source_result.stdout} should be contained"
                     f"in {available_clocksources_array}."
                 ).contains(current_clock_source_result.stdout)
+
+    @TestCaseMetadata(
+        description="""
+        This test is to check -
+            1. Current clock event name is 'Hyper-V clockevent'.
+            2. 'Hyper-V clockevent' and 'hrtimer_interrupt' show up times in
+             /proc/timer_list should equal to cpu count.
+            3. when cpu count is 1 and cpu type is not AMD type, unbind current time
+             clock event, check current time clock event switch to 'lapic'.
+        """,
+        priority=1,
+    )
+    def timesync_check_unbind_clockevent(self, node: Node) -> None:
+        if node.shell.exists(PurePosixPath(f"{self.current_clockevent}")):
+            # 1. Current clock event name is 'Hyper-V clockevent'.
+            clock_event_name = "Hyper-V clockevent"
+            event_handler_name = "hrtimer_interrupt"
+            cat = node.tools[Cat]
+            clock_event_result = cat.run(self.current_clockevent)
+            assert_that(clock_event_result.stdout).described_as(
+                f"Expected clockevent name is {clock_event_name}, "
```

Comment:
The actual and expected will be printed by assertpy. The description should explain why this needed, how to fix or the reference.